notebooks/official/workbench/spark/spark_sample_notebook.ipynb

```diff
@@ -0,0 +1,1200 @@
+{
```

Comment:
<p>Small suggestion to remove the "Note" and instead refactor the <strong>Dataset </strong>section to say</p><p> </p><p>The dataset you are using is the <a href="https://console.cloud.google.com/marketplace/product/github/github-repos" target="_blank">GitHub Activity Data</a>, available in <a href="https://cloud.google.com/bigquery/public-data" target="_blank">BigQuery Public Datasets</a>. We will refer to repositories that encompass polyglot programming as <strong>polyglot repos</strong> and those which only contain one programming language are referred to as <strong>monoglot repos</strong>. The first 1TB of data queried each month is free.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/720/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>